Mathlib/AlgebraicGeometry/Scheme.lean

```diff
@@ -704,8 +704,7 @@ lemma zeroLocus_map {U V : X.Opens} (i : U ≤ V) (s : Set Γ(X, V)) :
   ext x
   suffices (∀ f ∈ s, x ∈ U → x ∉ X.basicOpen f) ↔ x ∈ U → (∀ f ∈ s, x ∉ X.basicOpen f) by
     simpa [or_iff_not_imp_right]
-  conv_lhs => enter [i]; rw [forall_comm (β := x ∈ U)] -- why doesn't simp fire on this
-  rw [forall_comm (β := x ∈ U)]
+  grind
```

Comment:
The change looks good; I am just not sure if the comment should be tracked or it's simply not actionable.

Reply:
Now the comment is hard to understand in isolation: let's have another reviewer's opinion.